azurerm/internal/services/monitor/monitor_activity_log_alert_resource_test.go

```diff
@@ -160,6 +160,28 @@ func TestAccMonitorActivityLogAlert_basicAndCompleteUpdate(t *testing.T) {
 	})
 }
 
+func TestAccMonitorActivityLogAlert_basicServiceHealth(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_monitor_activity_log_alert", "test")
+	r := MonitorActivityLogAlertResource{}
+
+	data.ResourceTest(t, r, []resource.TestStep{
+		{
+			Config: r.serviceHealth(data),
+			Check: resource.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("enabled").HasValue("true"),
+				check.That(data.ResourceName).Key("description").HasValue(""),
+				check.That(data.ResourceName).Key("scopes.#").HasValue("1"),
+				check.That(data.ResourceName).Key("criteria.#").HasValue("1"),
+				check.That(data.ResourceName).Key("criteria.0.servicehealth.0.events.0").HasValue("Incident"),
+				check.That(data.ResourceName).Key("criteria.0.servicehealth.0.events.1").HasValue("Maintenance"),
+				check.That(data.ResourceName).Key("criteria.0.servicehealth.0.regions.0").HasValue("Global"),
+				check.That(data.ResourceName).Key("criteria.0.servicehealth.0.regions.1").HasValue("West Europe"),
```

Comment:
Actually in the test checkings, for the non-computed attributes, we do not need to check them one by one manually. The importStep will check all of them for you

Reply:
ok, thx I will clean the checks.